models/lido/liquidity/linea/lido_liquidity_linea_syncswap_pools.sql

```diff
@@ -0,0 +1,291 @@
+{{ config(
+    schema='lido_liquidity_linea',
+    alias = 'syncswap_pools',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'time'],
+    post_hook='{{ expose_spells(\'["linea"]\',
+                                "project",
+                                "lido_liquidity",
+                                \'["pipistrella"]\') }}'
```

Comment:
We may use the new form of `post_hook` here:
```suggestion
    post_hook='{{ expose_spells(blockchains  = \'["linea"]\',
                                spell_type   = "project",
                                spell_name   = "lido_liquidity",
                                contributors = \'["pipistrella"]\') }}'
```